azure-functions/acbs-function/mappings/facility/helpers/get-cover-start-date.js

```diff
@@ -2,19 +2,19 @@ const { formatDate } = require('../../../helpers/date');
 
 /**
  * Formatted EPOCH, if `fromEpoch` set to true else returns the input.
- * @param {String} epoch EPOCH
- * @param {Boolean} fromEpoch Format timestamp
- * @returns {String} Formatted timestamp if true
+ * @param {string} epoch EPOCH
+ * @param {boolean} fromEpoch Format timestamp
+ * @returns {string} Formatted timestamp if true
  */
 const format = (epoch, fromEpoch) => (fromEpoch ? formatDate(epoch) : epoch);
 
 /**
  * Evaluates the cover start date of a facility.
  * When the product is BSS/EWCS then requestedCoverStartDate
  * property is referred to else coverStartDate.
- * @param {Object} facility Deal's facility object
+ * @param {object} facility Deal's facility object
  * @param {Bool} formatted Return in YYYY-MM-DD date format
```

Comment:
Can all the `Bool` be updated to `boolean` please across all the files.

Reply:
updated, and caught a few more that slipped through